packages/angular/src/generators/component/files/__fileName__/__fileName__.__type__.ts__tpl__

```diff
@@ -0,0 +1,27 @@
+import { <% if(changeDetection !== 'Default') { %>ChangeDetectionStrategy, <% }%>Component<% if(!!viewEncapsulation) { %>, ViewEncapsulation<% }%> } from '@angular/core';<% if(standalone) {%>
+import { CommonModule } from '@angular/common';<% } %>
+
+@Component({<% if(!skipSelector) {%>
+  selector: '<%= selector %>',<%}%><% if(standalone) {%>
+  standalone: true,
+  imports: [CommonModule],<%}%><% if(inlineTemplate) { %>
+  template: `
+    <p>
+      <%= fileName %> works!
+    </p>
+  `<% } else { %>
+  templateUrl: './<%= fileName %><%= type ? '.' + type : '' %>.html'<% } if(inlineStyle) { %>,
```

Comment:
I guess if `style === 'none'` we don't want to generate inline styles either, isn't it?

Reply:
no, i think you need one or the other. this just matches what the Angular CLI generates right now anyway too